utilities/sim_close_app.scpt

```diff
@@ -0,0 +1,4 @@
+activate application "Simulator"
```

Comment:
Add some comments to indicate what this does

Reply:
As it self explanatory, it may be required to add comments. If you think it is still required, I can add

Reply:
Yes, a brief summary would be good.